src/e2eTest/tests/searchCase.spec.ts

```diff
@@ -0,0 +1,63 @@
+import {test} from '@playwright/test';
+import {parentSuite} from 'allure-js-commons';
+import configData from "@config/test.config";
+import {initIdamAuthToken, initServiceAuthToken,getUser} from '../utils/helpers/idam-helpers/idam.helper';
+import caseDataJson from 'data/case.data.json';
+import {
+    initializeExecutor,
+    performAction,
+    performValidation
+} from '@utils/controller';
+import {caseData} from '@data/case.data';
+import {getCaseInfo} from '@utils/actions/custom-actions/createCase.action';
+
+test.beforeEach(async ({page}, testInfo) => {
+    initializeExecutor(page);
+    await parentSuite('Case Creation');
+    await performAction('navigateToUrl', configData.manageCasesBaseURL);
+    await performAction('login', 'exuiUser');
+    const userCreds = getUser('exuiUser');
+    await initIdamAuthToken(userCreds?.email ?? '', userCreds?.password ?? '');
+    await initServiceAuthToken();
+    await testInfo.attach('Page URL', {
+        body: page.url(),
+        contentType: 'text/plain',
+    });
+    createCaseWithAddress()
```

Comment:
And this createCaseWithAddress can be moved to a beforeAll statement now that it doesn't depend on page anymore.